.circleci/config.yml

```
@@ -9,6 +9,10 @@ jobs:
       - image: perldocker/perl-tester:<< parameters.perl-version >>
     steps:
       - checkout
+      - run:
+          name: Install Redis server
+          command: |
+            apt-get install -y redis && apt-get install -y redis-server 
```

I guess you need install only `redis-server`
and if you want to install both , you can combine them:
```suggestion
            apt-get install -y redis-server redis 
```